misaligned.c

```diff
@@ -1,21 +1,35 @@
 #include <stdio.h>
 #include <assert.h>
 
+const char* majorColor[] = {"White", "Red", "Black", "Yellow", "Violet"};
+const char* minorColor[] = {"Blue", "Orange", "Green", "Brown", "Slate"};
+
 int printColorMap() {
-    const char* majorColor[] = {"White", "Red", "Black", "Yellow", "Violet"};
-    const char* minorColor[] = {"Blue", "Orange", "Green", "Brown", "Slate"};
     int i = 0, j = 0;
     for(i = 0; i < 5; i++) {
         for(j = 0; j < 5; j++) {
-            printf("%d | %s | %s\n", i * 5 + j, majorColor[i], minorColor[i]);
+            printf("%d | %s | %s\n", (i * 5 + j) + 1, majorColor[i], minorColor[j]);
         }
     }
     return i * j;
 }
 
+int printColorPair(int majorColorNumber, int minorColorNumber) 
+{
+    int i = majorColorNumber, j = minorColorNumber, pairNumber;
+    printf("ColorPair\n");
+    printf("%d | %s | %s\n", (i * 5 + j) + 1, majorColor[majorColorNumber], minorColor[minorColorNumber]);
+    pairNumber = (i * 5 + j) + 1;   
+    return pairNumber;
+}
+
 int main() {
+    int majorColorNumber = 1;
+    int minorColorNumber = 2;
     int result = printColorMap();
+    int pairNumber = printColorPair(majorColorNumber, minorColorNumber);
     assert(result == 25);
+    assert(pairNumber == 8);

```

Comment:
the tests and asserts are weak in nature. The alignment and the strings printed cannot be verified. Hence complete verification is not possible. Also misalignment due to single digit pair number and double digit pair number ex. 
0 | ...
12 | ...
is undetectable.

Please enhance tests to verify the output strings too.